thunder/dynamo/splitter.py

```diff
@@ -143,6 +144,8 @@ def is_thunder_supported_partition(node: torch.fx.Node) -> bool:
     for node in split_gm.graph.nodes:
         if is_thunder_supported_partition(node):
             graph_module = getattr(split_gm, node.name)
+            # Replace the torch operators within the function called by activation checkpoint with the corresponding Thunder symbols
```

Comment:
The original comment uses "operators" and "symbols" interchangeably. It's better to stick with one term in one sentence for clarity.
```suggestion
            # Replace PyTorch operators within the checkpointed function with the corresponding Thunder operators
```